pulseapi/entries/test_views.py

```diff
@@ -19,7 +19,7 @@ def test_post_minimum_entry(self):
         """
         Test posting an entry with minimum amount of content
         """
-        payload = self.generatePostPayload(data={'title':'title test_post_minimum_entry'})
+        payload = self.generatePostPayload(data={'title':'title test_post_minimum_entry', 'tags':''})
```

Comment:
We probably also want to make sure we have full test coverage for required fields now. https://github.com/mozilla/network-pulse-api/blob/fb8a57cff6fcd027bf9eb9766a71bd1ae5d7733f/pulseapi/entries/test_views.py#L38 tests for title, but with `content_url` also being required, we want to make sure that's a 400 as well?